documentation/docs/02_account_manager.md

```diff
@@ -81,22 +79,56 @@ Initialise the wallet with a randomly generated mnemonic and a given node.
 $ ./wallet init --node "http://localhost:14265/"
 ```
 
-## `./wallet new`
+### `./wallet new`
+
+Creates a new account.
+
+The wallet needs to be initialised (`init` command).
+
+#### Parameters
 
-(init first)
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `alias` | ✓         | Account index | "main"  |
 
-### Parameters
+#### Example(s)
 
-### Example(s)
+Create a new account with the account index as alias.
+```sh
+$ ./wallet new
+```
+
+Create a new account with a provided alias.
+```sh
+$ ./wallet new main
+```
 
-## `./wallet set-node`
+### `./wallet set-node`
 
-### Parameters
+Sets the node to be used for all requests.
 
-### Example(s)
+The new node URL is persisted to the storage and all future requests will use it.
 
-## `./wallet sync`
+#### Parameters
 
-### Parameters
+| Name  | Optional  | Example                   |
+| ----- | --------- | ------------------------- |
+| `url` | ✘         | "http://localhost:14265/" |
 
-### Example(s)
+#### Example(s)
+
+Set the node to be used for all requests.
+```sh
+$ ./wallet set-node http://localhost:14265/
+```
+
+### `./wallet sync`
+
+Synchronises all accounts.
+
+#### Example(s)
+
+Synchronise all accounts.
```

Comment:
Remove this line?

Reply:
Yeah I wanted consistency with all examples but I guess when there is only one I can ignore the description

Reply:
Done